Bio/PDB/PDBList.py

```diff
@@ -90,7 +110,7 @@ def get_status_list(self, url):
             for line in handle:
                 pdb = line.strip()
                 assert len(pdb) == 4
-                answer.append(pdb)
+                answer.append(pdb.decode())
```

Comment:
Does this work nicely on both Python 2 and 3? Would it be simpler to use ``from Bio._py3k import _as_string`` here?